dashboard/templates/grades.html

```diff
@@ -14,12 +14,12 @@
             <div class ="error"></div>
             </div>
         {% endblock %}
-        {% block js %}
-            <script src="{% static 'd3-v4/build/d3.min.js' %}"></script>
-            <script src="/static/js/grade_distribution.js"></script>
-        {% endblock %}
     {% else %}
         Grade Distribution view is hidden for this course.
     {% endif %}
 {% endblock %}
+{% block js %}
```

Comment:
not sure why this change is needed for current issue: if grade distribution is hidden, no js is needed, right?

Reply:
This is bug that resulted from JS error is the console  as this added the <script> tags twice in the html as resulted i guess from the  toggle view changes . So this is not related to current issue but I felt might as well fix it since I am here